### PR TITLE
Simple footer component

### PR DESCRIPTION
**Pull request by mbfitz**
I created a simple Footer component, including logo, copyright, nav links, social icons/links, and contact info. It uses flexbox for the layout and stacks at 669px (our "medium" mq). I also updated the components.scss to make sure they were in alphabetical order. This is set on footer.primary in the main .scss file. Here's a preview link: https://htmlpreview.github.io/?https://github.com/imarc/boilerplate/blob/2c732a26730dfeee54cd8c36067f827ee95f6e87/components/footer.html

### CODE REVIEW

File: css/_components.scss [hunk 1]
```diff
@@ -144,28 +144,85 @@ $large: 1024px;
...
+        a {
+            margin-bottom: 2rem;
+        }
+    }
+    .legal {
+        display: flex;
+        font-size: 0.8125rem;
+        a {
+            color: lighten($charcoal, 55%);
+            flex-basis: 25%;
```
Comment (TommyChanthaboune) [inline css/_components.scss]: I usually accompany `flex-basis` with 'max-width` for ie11 support

File: css/_components.scss [hunk 2]
```diff
@@ -144,28 +144,85 @@ $large: 1024px;
...
+        }
+    }
+    .copyright {
+        color: $meta-text;
+        font-size: 0.875rem;
+        padding-left: 0.5rem;
+        a {
+            color: lighten($charcoal, 55%);
+            flex-basis: 25%;
```
Comment (TommyChanthaboune) [inline css/_components.scss]: see previous comment about ie11 support

File: css/_components.scss [hunk 3]
```diff
@@ -144,28 +144,85 @@ $large: 1024px;
...
-    top: 0;
-    transition: top 0.2s ease-in-out;
-    width: 100%;
-    z-index: 100;
+    @media only screen and (min-width: $medium) {
+        text-align: left;
+        .container > div {
+            display: flex;
+            > div {
+                flex-basis: 33%; /* change this value for different # of columns */
```
Comment (TommyChanthaboune) [inline css/_components.scss]: see previous comment about ie11 support

**Review by TommyChanthaboune — COMMENTED** [review]
ie11 support

**Comment by marcelmoreau** [comment]
Was going to give comments but figured I would go in and make some changes since I am here. Not sure if this is how we want to do any of this moving forward or if it's too much stepping on the toes. I did not do much but:
- removed the flex-basis from the flex-items
- nav grows at twice the proportion that the other colums do
- flexbox turns on at desktop instead of tablet
- both the text and links in the footer were white so I made links look like links
- removed github social icon and made the icons slightly larger
- moved address to be near the logo which made more sense to me
- renamed `.copyright` to `.legal` as the latter can semantically encompass more content such as the legal links

Curious to hear Christian's take on our generic footer at the build-a-thon

**Comment by marcelmoreau** [comment]
This is simple. We can just merge this in and we all have the power to make more edits in the future. Can't see this one being a show-stopper.

**Pull request merged**